shell/app/config-page/components/table/render-types.tsx

```diff
@@ -320,7 +320,7 @@ export const getRender = (val: any, record: CP_TABLE.RowData, extra: any) => {
       }
       break;
     default:
-      Comp = val ? <Ellipsis title={`${val}`}>{`${val}`}</Ellipsis> : null;
+      Comp = val || val === 0 ? <Ellipsis title={`${val}`}>{`${val}`}</Ellipsis> : null;
```

Comment:
better to add a bracket to clarify the condition
`(val || val === 0) ? <Ellipsis title={`${val}`}>{`${val}`}</Ellipsis> : null;`